src/main/java/io/cdap/plugin/gcp/spanner/source/SpannerSource.java

```diff
@@ -104,7 +103,7 @@ public void configurePipeline(PipelineConfigurer pipelineConfigurer) {
     config.validate(collector);
     Schema configuredSchema = config.getSchema(collector);
 
-    if (!config.shouldConnect() || config.tryGetProject() == null
+    if (!config.canConnect() || config.tryGetProject() == null
```

Comment:
Can't put a comment at Line 81, can you add a Metadata annotation for the source like https://github.com/data-integrations/google-cloud/blob/develop/src/main/java/io/cdap/plugin/gcp/bigquery/source/BigQuerySource.java#L70?

Reply:
fixed